app/controllers/annotations_controller.rb

```diff
@@ -2,23 +2,14 @@
 
 # Generic resources controller
 class AnnotationsController < Annotot::AnnotationsController
-  before_action :load_and_authorize_project
+  prepend_before_action :load_and_authorize_project
```

Comment:
This is needed to run the filter before `Annotot::AnnotationsController` runs its filters. I believe beforehand, this was a no-op, which is why `load_and_authorize_project` needed to be hard-coded into the methods below.